docs/src/content/docs/utilities/add-virtual-import.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: addVirtualImport
+description: Allows adding Vite Virtual Modules easily to access data across your integration.
+---
+
+`addVirtualImport` allows you to create modules and store state across your integration. For example:
```

Comment:
```suggestion
`addVirtualImport` allows you to create virtual modules and share data from your integration. For example:
```

Reply:
Can you also adds a sentence about conventions? I'm thinking about 2 things specifically:
1. If the module is considered internal, follow this pattern: `virtual:${name}`
2. If the module is meant for users' usage, follow this pattern `${integrationName}:${name}`